gcloud-java-examples/src/main/java/com/google/gcloud/examples/Stat.java

```diff
@@ -0,0 +1,104 @@
+package com.google.gcloud.examples;
```

Comment:
Thank you! I moved it, updated the README, checked that the command-line there works.